### PR TITLE
Update article about pointing domain to Heroku

### LINKED ISSUE #139: Update point domain apex to Heroku article
This article http://support.dnsimple.com/articles/domain-apex-heroku/ has old record editor screenshots that are easy enough to update.

### PR DESCRIPTION
**Pull request by jacegu**
Closes #139 

I have updated pretty much everything and added a section about configuring the custom domain on Heroku which was surprisingly missing.

Note that there is a link to an article (the one click services one) that is still in a PR.

### CODE REVIEW

File: content/articles/domain-apex-heroku.markdown [hunk 1]
```diff
@@ -1,33 +1,63 @@
 ---
 title: Pointing the Domain Apex to Heroku
-excerpt: Welcome to DNSimple. This page is about pointing the domain apex at Heroku. Hosted DNS has never been this easy.
+excerpt: How to point your domain to your Heroku application using DNSimple.
```
Comment (weppos) [inline content/articles/domain-apex-heroku.markdown]: :heart:

File: content/articles/domain-apex-heroku.markdown [hunk 2]
```diff
@@ -1,33 +1,63 @@
...
+
 Your domain apex is the second or third level domain provided to you by your domain registrar. For example, `dnsimple.com` is the apex domain for DNSimple.
 
-To point your apex to Heroku you have 2 possibilities:
+To point your apex to Heroku you have 2 options:
 
-- Use the Heroku one-click service (for non-SSL usage)
-- Add an [ALIAS record](/articles/alias-record/) pointing your custom Heroku end point (for non-SSL and SSL usage)
+1. Using the Heroku one-click service.
+1. Manually adding an [ALIAS record](/articles/alias-record/).
```
Comment (weppos) [inline content/articles/domain-apex-heroku.markdown]: We may need some :us: :person_with_blond_hair: here, but I prefer the non-ing version

- Use ...
- Manually add...

File: content/articles/domain-apex-heroku.markdown [hunk 3]
```diff
@@ -1,33 +1,63 @@
...
-## Creating an ALIAS for the apex domain
 
-Go to the advanced editor page for your domain to manage your DNS records. Click *Add a Record* and select the *ALIAS* record type.
+Once your apex domain is pointing to correct place you will have to configure the custom domain at Heroku.
 
-![Add a Record](/files/dnsimple-alias-record.png)
+## Using the Heroku one-click service
 
-In the form enter the following settings:
+You can only use the one-click service if you are not using SSL on your Heroku application. If you are please follow the instructions in [the next section](#point-using-alias).
```
Comment (weppos) [inline content/articles/domain-apex-heroku.markdown]: We do have a [one-click Heroku SSL service](https://github.com/aetrion/dnsimple-services/blob/master/services/heroku-ssl/readme.md).

I'd keep the manual instructions, but we should mention that the user can use the right one-click service depending on whether he/she's using SSL or not.

**Comment by jacegu** [comment]
@weppos thanks for the feedback. I've updated the article accordingly.

**Comment by weppos** [comment]
:sheep: :it:

**Comment by jacegu** [comment]
I'll wait for the services article to be deployed since otherwise I'd deploy a broken link.

**Pull request merged**